### DISCUSSION TITLE
Tesla not connecting to Tesla-Android wifi

### DISCUSSION THREAD

**Discussion by garethmarlow** [tesla-android/issue-tracker#77] [Q&A]
My car is refusing to connect to the Tesla-Android wifi. I get the error messages in the attached images:

"Unable to obtain IP address"

and then on retry

"This network tried to present a login page..."

Car is running 2022.20.7

Any ideas folks
![IMG_3634](https://user-images.githubusercontent.com/59932701/182337643-a07746cb-98c3-4998-84d1-54fa0c085a0c.jpeg)
![IMG_3635](https://user-images.githubusercontent.com/59932701/182337674-a28a057e-b4d3-4a7c-968a-1f89e97f0e32.jpeg)
?

**Comment by Mateusnasciment** [comment]
have EXACTLY the same issue on my Model Y. Rebooting the car solves the issues, but when I leave the house and come back, it will not see or reconnect tot eh network. I also have a TP-Link setup, and wondering if there is some sort of settings issue between the Tesla and the TP-Link router, and perhaps a recent update for one of the two caused these issues.
